src/Api/Endpoints/Events.cs

```diff
@@ -0,0 +1,37 @@
+using ApiHelpers;
+using Passwordless.Api.Authorization;
+using Passwordless.Service.AuditLog.Loggers;
+using Passwordless.Service.AuditLog.Mappings;
+using Passwordless.Service.Features;
+
+namespace Passwordless.Server.Endpoints;
+
+public static class AuditLog
+{
+    public static void MapAuditLogEndpoints(this WebApplication app)
+    {
+        app.MapGet("events", GetAuditLogEvents)
+            .RequireSecretKey()
+            .RequireCors("default");
+    }
+
+    private static async Task<IResult> GetAuditLogEvents(int pageNumber,
+        int numberOfResults,
```

Comment:
We should provide default and max value for numberofResults. Recommended default = 100 and max = 1000. For max we just cap it to 1000, no errors thrown if you go above it.